shuup/core/templatetags/shuup_common.py

```diff
@@ -114,6 +114,9 @@ def datetime(value, kind="datetime", format="medium", tz=True):
 
     locale = get_current_babel_locale()
 
+    if value is None:
```

Comment:
Ok, will do. This is originally from @gurch101 branch. I will see if there is really use for this.